pages/index.js

```diff
@@ -1,131 +1,160 @@
-import Head from 'next/head'
-import { useState, useRef } from 'react'
-
-import axios from 'axios'
-import domtoimage from 'dom-to-image'
-import { saveAs } from 'file-saver'
-import Header from '../components/Header'
-import Main from '../components/Main'
-import Settings from '../components/Settings'
-
-import {Text, Box, Flex} from "@chakra-ui/react"
-
-
-function App() 
-{
-    
-    const [bg, setBg] = useState('linear-gradient(106.8deg, rgb(117, 255, 220) 6%, rgb(163, 216, 255) 47.6%, rgb(248, 215, 251) 87.8%)')
-
-    const tweetRef = useRef(null)
-
-    const [tweetData, setTweetData] = useState(null)
-
-    const [showTime, setShowTime] = useState(true)
-    const [showMetrics, setShowMetrics] = useState(true)
-    const [showSource, setShowSource] = useState(true)
-    
-    const [scale, setScale] = useState(0.9)
-
-    const [hint, setHint] = useState(true)
-    const [loading, setLoading] = useState(false)
-    const [error, setError] = useState(false)
-
-    const bringTweet = async (e) => {
-        try {
-            e.preventDefault()
-            setHint(false)
-            setLoading(true)
-            const url = e.target.elements.tweetURL.value
-            const id = url.split('/')[5]
-            const { data, status } = await axios.get(`/api/tweet/${id}`)
-            if(data.message){
-                setError(true)
-                setLoading(false)
-                setTweetData(null)
-            }
-            else {
-                setLoading(false)
-                setTweetData(data.data)
-                setError(false)
-            }
-        } catch (e) {
-            setError(true)
-            setLoading(false)
-            setTweetData(null)
-        }
+import Head from "next/head";
+import { useState, useRef } from "react";
```

Comment:
There are too many changes in this file that are a result of running a prettier. Because this project is not using prettier, please disable it, and make your changes again before we can continue review process